apis/python/src/tiledbsoma/_soma_object.py

```diff
@@ -38,20 +32,10 @@ class SOMAObject(somacore.SOMAObject, Generic[_WrapperType_co]):
         Maturing.
     """
 
-    _wrapper_type: (
-        type[_WrapperType_co]
-        | type[_tdb_handles.DataFrameWrapper]
-        | type[_tdb_handles.DenseNDArrayWrapper]
-        | type[_tdb_handles.SparseNDArrayWrapper]
-        | type[_tdb_handles.CollectionWrapper]
-        | type[_tdb_handles.ExperimentWrapper]
-        | type[_tdb_handles.MeasurementWrapper]
-        | type[_tdb_handles.SceneWrapper]
-        | type[_tdb_handles.MultiscaleImageWrapper]
-    )
-    """Class variable of the Wrapper class used to open this object type."""
-
-    __slots__ = ("_close_stack", "_handle", "_handle_wrapper")
+    _handle_type: _tdb_handles.RawHandle
```

Comment:
Class variables should be typed as such (for clarity and to help toolchain). In this case:

```
_handle_type: ClassVar[_tdb_handles.RawHandle]
```